app/Http/Controllers/Auth/LoginController.php

```diff
@@ -43,7 +43,7 @@ public function __construct()
      * @param  Request $request
      * @return Response
      */
-    public function logout(Request $request): Response
```

Comment:
What error did you get? It should work, so getting an error is weird behaviour.

Reply:
Seeing the returned object is a `RedirectResponse` which does not extend the `Illuminate\Http\Response` the hinted return type is invalid. Which means the docBlock is also wrong.

I'd suggest we go for either hinting it with:

* `Illuminate\Http\RedirectResponse`, verbose and clear, but limiting
* `Symfony\Component\HttpFoundation\Response`, less verbose but we can return any type of response

Both are fine, pick what you think suits the situation best.

Reply:
I think I also need to change :Reponse to :RedirectRespose. is that right?

Reply:
and also that one above it from `@return Response` to `@return RedirectReponse`

Reply:
si